docs/spec/spec.md

```diff
@@ -3,14 +3,14 @@
 _Owners_: @daneshk @niveathika  
 _Reviewers_: @daneshk  
 _Created_: 2022/01/14   
-_Updated_: 2022/01/14  
+_Updated_: 2022/01/25  
 _Issue_: [#2290](https://github.com/ballerina-platform/ballerina-standard-library/issues/2290)
 
 # Introduction
 
-This is the specification for JDBC standard library, which provides the functionality that is required to access and 
-manipulate data stored in any type of relational database, which is accessible via Java Database Connectivity (JDBC) 
-in the [Ballerina programming language](https://ballerina.io/), which is an open-source programming
+This is the specification for the JDBC standard library, which provides the functionality that is required to access and 
```

Comment:
```suggestion
This is the specification for the JDBC standard library, which provides the functionality that is required to access and 
```
```suggestion
This is the specification for the JDBC standard library, which provides the functionality that is required to access and 
```
This sentence is too long. Consider breaking it into multiple  sentences.

Reply:
Fixed in fc94727